### ISSUE TITLE
Tab Key is invalid !!!

### ISSUE THREAD

**Issue by achillis2016** [ipython/ipython#12555]
Tab key is very very useful when i use ipython, but I found it does not work with ipython version=7.18.1 , when I press Tab key, nothing happen.So i tested some previous version, this problem may appear after version 7.18.0

My Envirement: Win7 x64 SP1, python 3.8.2 amd64

**Comment by hehelego** [comment]
I have got the same issue.  

> python 3.8.5
> ipython 7.18.1


> screenfetch

```plaintext
 OS: Manjaro 20.1.1 Mikah
 Kernel: x86_64 Linux 5.8.11-1-MANJARO
 Shell: fish 3.1.2
 CPU: AMD Ryzen 7 4800U with Radeon Graphics @ 16x 1.8GHz
 GPU: AMD RENOIR (DRM 3.38.0, 5.8.11-1-MANJARO, LLVM 10.0.1)
```

**Comment by t1anchen** [comment]
same issue here. Tab completion not working in 7.18.+

- python 3.7.7
- ipython 7.18.1

A possible relevant issue might be around https://github.com/ipython/ipython/issues/12522

Temporary Mitigation: I removed ipython and downgraded to `7.17.0` with re-installing all dependencies to ipython and it worked back.

**Comment by l3shi1L** [comment]
Just downgrade jedi module to version 0.17.2
`pip install ipython
pip install  jedi==0.17.2`
Tested python3.6 python3.7 python3.8 in Win10.
Maybe works for you.

**Comment by MrMino** [comment]
This has been fixed in #12793.
